main_test.go

```diff
@@ -196,6 +196,26 @@ const emptyConfig = `
   }
 `
 
+const badInnternalMetrics = `
+  {
+    "ListenFrom":[
+    ],
+    "ForwardTo":[
+    ],
+    "InternalMetrics": "0.0.0.0:999999"
+  }
+`
+
+const internalMetrics = `
+  {
+    "ListenFrom":[
+    ],
+    "ForwardTo":[
+    ],
+    "InternalMetrics": "0.0.0.0:0"
```

Comment:
Port 0 is valid I guess but it will be hard to know what port the kernel chooses since it never logs it out.

Reply:
That's that point. If i hard code it it's possible the test will fail.